### PR TITLE
feature: better UI on mobile device

### PR DESCRIPTION
**Pull request by Jack-Works**
Mobile device:

image

image

### CODE REVIEW

File: src/extension/background-script/WelcomeService.ts [hunk 1]
```diff
@@ -228,3 +228,8 @@ export async function openWelcomePage(id?: SocialNetworkUI['lastRecognizedIdenti
     }
     return browser.tabs.create({ url: getWelcomePageURL(id) })
 }
+
+export async function openOptionsPage(url: string) {
```
Comment (Artoria2e5) [inline src/extension/background-script/WelcomeService.ts]: openExtensionPage?

File: src/index.tsx [hunk 2]
```diff
@@ -167,39 +148,56 @@ function ResponsiveDrawer() {
...
                 </div>
             </Router>
         </ThemeProvider>
     )
 }
-function LinkItem(props: { to: string; icon: React.ReactElement; title: string }) {
+function MobileBackButton() {
+    if (!(webpackEnv.firefoxVariant === 'GeckoView' || webpackEnv.target === 'WKWebview')) return null
     return (
+        <MuiLink color="textPrimary" onClick={window.close}>
```
Comment (Artoria2e5) [inline src/index.tsx]: Is that a close button? Use a "close" button if your back button is really intended to close the popup, or use `window.history.back` for back. See the semantics at [material.io](https://material.io/design/navigation/understanding-navigation.html).

The spacing on that button looks awkward. Specifically:
* There should be more space to the right of the arrow. The placement should be identical to the `menuButton` element in index.ts.
* The baseline of the logo looks off. Should be fixed by applying the `menuButton` margin: as long as it's not directly next to the IconButton it should be fine.

File: src/social-network-provider/facebook.com/worker-provider.ts [hunk 3]
```diff
@@ -9,6 +9,7 @@ export const facebookWorkerSelf = defineSocialNetworkWorker({
     ...sharedProvider,
     fetchPostContent: fetchPostContentFacebook,
     fetchProfile: fetchProfileFacebook,
-    autoVerifyBio: autoVerifyBioFacebook,
+    autoVerifyBio: webpackEnv.target === 'WKWebview' ? null : autoVerifyBioFacebook,
```
Comment (Artoria2e5) [inline src/social-network-provider/facebook.com/worker-provider.ts]: ……okay.

File: src/extension/background-script/WelcomeService.ts (hunk 1 above)
Comment (Jack-Works) [reply src/extension/background-script/WelcomeService.ts]: they are synonyms since we don't expect user to open background page or popup page

**Comment by Artoria2e5** [comment]
#288, but it should be non-blocking.

File: src/index.tsx (hunk 2 above)
Comment (Jack-Works) [reply src/index.tsx]: this button only appears on Android or iOS standalone app. there is no "tabs" equivalent in those environment so clicking on the back button behaves like "back" from the settings page to the Facebook.com

File: src/extension/background-script/WelcomeService.ts (hunk 1 above)
Comment (Artoria2e5) [reply src/extension/background-script/WelcomeService.ts]: `(url: string)`

**Comment by Jack-Works** [comment]
> #288, but it should be non-blocking.

you can change from request change to approve

File: src/extension/background-script/WelcomeService.ts (hunk 1 above)
Comment (Jack-Works) [reply src/extension/background-script/WelcomeService.ts]: maybe `/index.html#/welcome` or something. ok I'll change that

**Comment by Artoria2e5** [comment]
2em.

**Comment by neruthes** [comment]
Please update UI according to https://github.com/DimensionDev/Maskbook_MobilePWA-Meta/issues/1

**Pull request merged**